src/auto_code_surveys.py

```diff
@@ -31,16 +31,43 @@ def auto_code_surveys(cls, user, data, phone_uuid_table, coda_output_dir):
             if plan.cleaner is not None:
                 CleaningUtils.apply_cleaner_to_traced_data_iterable(user, data, plan.raw_field, plan.coded_field,
                                                                     plan.cleaner, plan.code_scheme)
+
+        # For any locations where the cleaners assigned a code to a constituency, set the county code to NC
```

Comment:
Given that we're not applying cleaners to the locations, is this block needed?

Reply:
No was just getting ready for tests for other Kenyan based project incase we get time to build the cleaners. Happy to revert to previous flow and submit this as a pr later on.